packages/components-native/src/InputDate/InputDate.test.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react";
+import { fireEvent, render, waitFor } from "@testing-library/react-native";
+import { Host } from "react-native-portalize";
+import { FormProvider, useForm } from "react-hook-form";
+import { InputDate } from "./InputDate";
+import { Button } from "../Button";
+
+describe("InputDate", () => {
+  describe("Visuals", () => {
```

Comment:
It looks like there isn't a test for the formattedDate following the provided value from the AtlantisContext. Did I miss it?

Reply:
New tests were added for all 4 possible dateFormat patterns 👍